src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx

```diff
@@ -59,7 +52,16 @@ export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = ({ onCo
           )}
           <div className="text-base text-gray-800 my-4">{errorMessage || error ? "Please try again." : "or"}</div>
           <Button className="bg-cerulean text-white hover:bg-cerulean-500 border-gray-300 px-12">Browse Files</Button>
-        </div>
+          <a
+            onClick={(e) => e.stopPropagation()}
```

Comment:
can omit?

Reply:
cannot, cos the link is in the button i.e. clicking the link will cause redirect and cause the upload window to show

Reply:
ah okay got it